.ci/deps.sh

```diff
@@ -34,7 +34,8 @@ npm install
 mkdir -p ~/.RLibrary
 echo '.libPaths( c( "~/.RLibrary", .libPaths()) )' >> .Rprofile
 echo 'options(repos=structure(c(CRAN="http://cran.rstudio.com")))' >> .Rprofile
-R -e "install.packages('lintr', dependencies=TRUE, quiet=TRUE, verbose=FALSE)"
+R -e 'install.packages("lintr", dependencies=TRUE, quiet=TRUE, verbose=FALSE)'
+R -e 'install.packages("formatR", dependencies=TRUE, quiet=TRUE, verbose=FALSE)'
```

Comment:
the commit message/shortlog gives no information about what is done on it. we want commits to be clear about what the change does and why it is needed. trust me, helps a lot when going through old commits